les/peer.go

```diff
@@ -820,7 +820,7 @@ func (p *clientPeer) sendStop() error {
 
 // sendResume notifies the client about getting out of frozen state
 func (p *clientPeer) sendResume(bv uint64) error {
-	return p2p.Send(p.rw, ResumeMsg, bv)
+	return p2p.SendItems(p.rw, ResumeMsg, bv)
```

Comment:
This change should be reverted because it's a protocol change. 

The bug here is: Resume messages are encoded as `0x17 | bv` (no list for message content), but the spec says they should be encoded as `0x17 | [ bv ]` (with list around bv). Fixing this needs a workaround in the client because existing servers send the wrong encoding.